app/views/shared/_slotpicker_templates.html.erb

```diff
@@ -0,0 +1,76 @@
+<script type="text/html" id="SlotPicker-tmplDay">
+  <li class="SlotPicker-day" id="date-{{ slot }}" tabindex="-1">
+    <h2 class="SlotPicker-dayTitle">{{ date }}</h2>
+    {{#if oneSlot}}
+    <p class="SlotPicker-dayOneSlot"><%= t('steps.slotpicker_templates.one_slot_available') %></p>
```

Comment:
Not sure why, but it was refusing to find just `t('.one_slot_avaiable')` when it was included in `views.yml`.